frontend/packages/shared/src/styles/abstracts/_colors.scss

```diff
@@ -0,0 +1,19 @@
+$color-background: #f0f3f7;
+$color-primary: #ffffff;
+
+$color-divider: #e0e0e0; // not in figma specs
+
+$color-text-primary: #000a48;
+$color-secondary-dark: #000a48;
+$color-secondary: #0041dc;
+
+$color-grey-200: #f5f5f5;
+$color-grey-400: #cccccc;
+$color-grey-600: #999999;
+$color-grey-700: #666666;
+
+$color-blue-200: #f2f5fd;
+
+$color-red-500: #db2828;
+
+$color-green-600: #5fc82b;
```

Comment:
Figmassa näiden lisäksi jonkun verran muitakin värejä, mutta niitä ei ilmeisesti käytetä vielä mihinkään?

Reply:
Ei taida sovellus tosiaan käyttää muita värejä kuin näitä. Löysin `_typography.scss` alta lisäksi sellaisen kuin `color-disabled`. Ko. "väri" määrittää käytännössä saman kuin `opacity: 38%` joka vastaa material ui:n disabled väriä/läpinäkyvyyttä ja tuota käytetään `CustomSwitch`:ssä ihan suht fiksusti.